src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/ProjectServices/VsManagedLanguagesProjectSystemServices.cs

```diff
@@ -230,6 +230,12 @@ private static VersionRange ToVersionRange(string version, bool isCpvmEnabled)
                 // Projects that have their packages managed centrally will not have Version metadata on PackageReference items.
                 return null;
             }
+            else if (string.IsNullOrEmpty(version))
+            {
+                // https://github.com/NuGet/Home/issues/9289
```

Comment:
Only add URLs into the source to track work that needs to be done in the future. We don't want every line of code commented with which issue it was added for.